agents/mysql/perfschema/perfschema.go

```diff
@@ -206,8 +208,38 @@ func (m *PerfSchema) runHistoryCacheRefresher(ctx context.Context) {
 	}
 }
 
+func getMySQLVersion(q *reform.Querier) (mysqlVersion float64, vendor string, err error) {
```

Comment:
Just a comment. 
Usually we don't hve named return parameters in our functions. Maybe it is a topic to discuss.